proto/prysm/v1alpha1/generated.ssz.go

```diff
@@ -16,7 +16,7 @@ func (a *Attestation) MarshalSSZ() ([]byte, error) {
 // MarshalSSZTo ssz marshals the Attestation object to a target array
 func (a *Attestation) MarshalSSZTo(buf []byte) (dst []byte, err error) {
 	dst = buf
-	offset := int(228)
```

Comment:
this is generated, if we use fastssz to regenerate the marshalling paths its going to remove all these changes